packages/governance/src/question.js

```diff
@@ -90,23 +99,31 @@ const looksLikeIssueForType = (electionType, issue) => {
   switch (electionType) {
     case ElectionType.SURVEY:
     case ElectionType.ELECTION:
-      looksLikeSimpleIssue(/** @type {SimpleIssue} */ (issue));
+      assertSimpleIssue(/** @type {SimpleIssue} */ (issue));
       break;
     case ElectionType.PARAM_CHANGE:
-      looksLikeParamChangeIssue(/** @type {ParamChangeIssue} */ (issue));
+      assertParamChangeIssue(/** @type {ParamChangeIssue} */ (issue));
+      break;
+    case ElectionType.API_INVOCATION:
+      assertApiInvocation(/** @type {ApiInvocationIssue} */ (issue));
       break;
     default:
       throw Error(`Election type unrecognized`);
   }
+  return undefined;
```

Comment:
ditto as above.

if this is to make eslint happy, eslint has a bug in a rule which should be suppressed